<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKDebuggerDeviceMask</class>
 <widget class="QDialog" name="SAKDebuggerDeviceMask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>67</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mask Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Tx mask</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Rx mask</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="rxMaskCheckBox">
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="txMaskCheckBox">
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="txMaskSpinBox"/>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="rxMaskSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: &quot;red&quot;
}</string>
     </property>
     <property name="text">
      <string>Only supported one bytes mask yet!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
